<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comboBox</class>
 <widget class="QWidget" name="comboBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>comboBox</string>
  </property>
  <widget class="QLabel" name="label_showPixmap">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="hasDefaultTextEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_showList">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>hasDefaultTextEdit</class>
   <extends>QLineEdit</extends>
   <header>hasDefaultTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
